courses/free-courses-en.md

```diff
@@ -726,6 +727,11 @@
 * [Intro to Theoretical Computer Science](https://www.udacity.com/course/intro-to-theoretical-computer-science--cs313) (Udacity)
 
 
+### TwinCAT 3
+
+* [PLC programming using TwinCAT 3](https://www.youtube.com/playlist?list=PLimaF0nZKYHz3I3kFP4myaAYjmYk1SowO) - Jakob Sagatowski
```

Comment:
Provide platform

```suggestion
* [PLC programming using TwinCAT 3](https://www.youtube.com/playlist?list=PLimaF0nZKYHz3I3kFP4myaAYjmYk1SowO) - Jakob Sagatowski (Youtube)
```

Reply:
Good one. Could you just please correct the suggestion from Youtube -> YouTube.?

Reply:
Merelly click on accept code suggestion and will be autocommited to your pr 🤗

Reply:
Yes but then it will be spelled "Youtube" and not "YouTube"... :-)

Reply:
Changed. thanks. Try now